docs/tests/integration/test_wazuh_db/test_wazuh_db.md

```diff
@@ -0,0 +1,61 @@
+# Test Wazuh DB
+
+All these tests are meant to send a specific command to the socket (valid or not) and compare the resulting output with the expected result.
+## Overview
```

Comment:
Shouldn't we leave a white line between the previous paragraph and this title?

Reply:
Fixed!